sdk/pinocchio/src/lib.rs

```diff
@@ -239,6 +238,10 @@ pub mod sysvars;
 #[deprecated(since = "0.7.0", note = "Use the `entrypoint` module instead")]
 pub use entrypoint::lazy as lazy_entrypoint;
 
+// Re-export the `solana_account_view` for downstream use.
+pub use solana_account_view as account;
```

Comment:
nit: how about naming the re-export `account_view`?
```suggestion
pub use solana_account_view as account_view;
```

Reply:
I was meant to ask about this. At the moment we have:
* `program_error` -> `error`
* `solana_address` -> `address`

Then we could have:
* `account_view` -> `account`
* `instruction_view` -> `instruction`

It is a bit shorter if we "drop" the `_view`, but happy either way.